core-services/prow/02_config/_plugins.yaml

```diff
@@ -491,6 +495,15 @@ bugzilla:
       target_release: 4.6.z
       validate_by_default: true
   orgs:
+    jianzhangbjz:
+      repos:
+        learn-operator:
+          branches:
+            '*':
+              allowed_groups:
+              - redhat
+              - qe_staff
+              validate_by_default: false
```

Comment:
Do you expect to interact with RH BZ from your personal repository? That does not seem to be correct.

Reply:
Also, you do not seem to  enable teh `bugzilla` plugin (correctly, IMO), so this config should not be here.

Reply:
No, I don't want to interact with RH BZ. So, should I remove this `validate_by_default: false`?

Reply:
No, remove all config for BZ plugin

Reply:
Yes, thanks removed all of them.